### PR TITLE
Add items to read me

### PR DESCRIPTION
**Pull request by snprani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # Sample.ForkPull
 Sample repository for Fork and Pull model of Git-based collaboration
+
```
Comment (tonysneed) [inline README.md:3]: These should be numbered.

Comment (snprani) [reply README.md:3]: Oh good idea to have it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+1. Items 1
+1. Items 2
+1. Items 3
```
Comment (tonysneed) [inline README.md:6]: Let's make this 4 items.

Comment (snprani) [reply README.md:6]: Exactly.

**Comment by tonysneed** [comment]
This looks good. Let's ship it.
Go ahead and squash your commits.

**Pull request merged**